ext4/tar2ext4/tar2ext4_test.go

```diff
@@ -162,3 +165,76 @@ func Test_TarHardlinkToSymlink(t *testing.T) {
 		t.Fatalf("failed to convert tar to layer vhd: %s", err)
 	}
 }
+
+func calcExt4Sha256(t *testing.T, layerTar *os.File) string {
+	t.Helper()
+	if _, err := layerTar.Seek(0, 0); err != nil {
+		t.Fatalf("failed to seek file: %s", err)
+	}
+
+	opts := []Option{ConvertWhiteout}
+
+	tmpVhdPath := filepath.Join(os.TempDir(), "test-vhd.ext4")
```

Comment:
Minor nit: its not a VHD, since there is no VHD footer option

Reply:
Fixed in [6dee921](https://github.com/microsoft/hcsshim/pull/2270/commits/6dee921a81385e3338f40e6e78b0462f3f4d854e)